### PR TITLE
kill identical multiqc before running it

### PR DESCRIPTION
**Pull request by Galithil**
Since we have lingering multiqcs on large projects, this should help alleviate the issue.

### CODE REVIEW

File: ngi_pipeline/utils/post_analysis.py [hunk 1]
```diff
@@ -14,6 +14,14 @@ def run_multiqc(base_path, project_id, project_name, wait=False):
     multiqc_stdout=''
     multiqc_stderr=''
     try:
+        #if multiqc is already running, kill it first.
+        ps_command=["ps", "ux"]
+        pcs=subprocess.check_output(ps_command)
+        for line in pcs.splitlines():
+            if " ".join(command) in line :
+                os.kill(int(line.split()[1]), 9)
```
Comment (b97pla) [inline ngi_pipeline/utils/post_analysis.py]: since multiqc is not launched, I guess this code can go away?

Comment (vezzi) [inline ngi_pipeline/utils/post_analysis.py]: @Galithil can you remove this code.... at this point it is pointless

Comment (Galithil) [inline ngi_pipeline/utils/post_analysis.py]: The whole file has to go, then, is that okay ?

Comment (vezzi) [inline ngi_pipeline/utils/post_analysis.py]: yes, silence the execution of multiQC, we do not want it to start automatically

**Pull request closed without merging**